hedera-node/test-clients/scripts/diff-testing/configuration.py

```diff
@@ -0,0 +1,36 @@
+from dataclasses import dataclass
+from datetime import datetime
+
+
+@dataclass
+class Configuration:
+    # paths and filenames
+    LOCAL_STORAGE_ROOT: str = '/users/user/new-state/'  # where the states and streams will be stored
+    MAINNET_HOSTNAMES_FILENAME: str = 'mainnet-hostnames-2024-02-04.txt'  # file containing node# <-> hostname map
+    MAINNET_BLOBLIST_FILENAME: str = 'bloblist.txt'  # file for ongoing state (blobs to fetch, blobs fetched, etc.)
+
+    # interval to pull - start and end must _not_ cross 00:00Z (i.e., must be on same GMT day, except can have 00:00Z
+    # at either end)
+    INTERVAL_START_TIME: datetime = datetime.fromisoformat('2024-02-01T00:00:00')
+    INTERVAL_END_TIME: datetime = datetime.fromisoformat('2024-02-01T03:00:00')  # '2024-02-02T00:00:00')
+
+    # nodes
+    N_NODES: int = 29
+    DEFAULT_NODE_NUMBER: int = 5
+
+    # gcp buckets
+    GCP_PROJECT: str = 'mainnet-exports'
```

Comment:
Are we allowed to put this in the repo??

Reply:
GCP project name is public - you can't get to the buckets in the gcp dashboard without it.  The GCP _billing_ name is probably something to keep private - and it isn't in here - though even that can't really be abused AFAIK because they're access lists.